int8 ec_id          # ethercat板子的索引
int8 slave_id       # ethercat板子的子板的索引
int8 passage        # 通道号
int8 motor_id       # 电机id
float32 position    # 电机位置
float32 speed       # 电机速度
float32 current     # 电机电流
int8 ack_status     # 返回状态

